### ISSUE TITLE
Extra left brace produced in an inheritance declaration 

### ISSUE THREAD

**Issue by quirp** [RyuuGan/sol-merger#73]
It seems when there is no space between the rightmost inherited contract and the left brace, a double left brace will be produced in the output. 
```pragma solidity ^0.8.9;

contract B{   
}
contract A is B{
}

```
produces the following output when running the merge method  (```const { merge } = require('sol-merger');```),
```pragma solidity ^0.8.9;


contract B {   
}

contract A is B{{
}
```
Where we see an extra left brace added at contract A. 

Note these two scenarios do not produce extra left braces: 

1. At least one space in between the left brace and the rightmost inherited contract
```pragma solidity ^0.8.9;

contract B{   
}
contract A is B {
}
```
2. No space needed between contract variable name and the left brace.

```pragma solidity ^0.8.9;

contract A{
}


```

sol-merger@4.4.0

**Comment by RyuuGan** [comment]
Hi @quirp,

Thanks for the issue.

Will take a look soon.

Best regards,
Valerii Aligorskii

**Comment by RyuuGan** [comment]
Fix is available in npm [v4.4.1](https://www.npmjs.com/package/sol-merger/v/4.4.1)
